homeassistant/components/ecoal_boiler/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""
+Support for ecoal/esterownik.pl coal/wood boiler controller.
+
+Allows read various readings available in controller
+and set very basic switches.
+"""
+import logging
+
+# import voluptuous as vol
+# import homeassistant.helpers.config_validation as cv
+
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = "ecoal_boiler"
+
+CONF_HOSTNAME = "host"
+CONF_LOGIN = "login"
+DEFAULT_LOGIN = "admin"
+CONF_PASSWORD = "password"
+DEFAULT_PASSWORD = "admin"
+
+
+# CONFIG_SCHEMA = vol.Schema({
+#    DOMAIN: vol.Schema({
+#        vol.Required(CONF_HOSTNAME): cv.string,
+#        vol.Optional(CONF_LOGIN,
+#                        default=DEFAULT_LOGIN): cv.string,
+#        vol.Optional(CONF_PASSWORD,
+#                        default=DEFAULT_PASSWORD): cv.string,
+#    })
+# })
+# Fails with:
+# Invalid config for [ecoal_boiler]:
+#   [homeassistant] is an invalid option for [ecoal_boiler].
+
+# In fact it is variable, set during setup
+ECOAL_CONTR = None
+
+
+async def async_setup(hass, config):
+    """Set up global ECOAL_CONTR same for sensors and switches."""
+    global ECOAL_CONTR
+    _LOGGER.debug("async_setup(): config: %r", config)
```

Comment:
Doesn't look useful for a user. Reword it or remove it.

Reply:
Removed.